GoogleDataTransportCCTSupport/GDTCCTTests/Unit/GDTCCTNanopbHelpersTest.m

```diff
@@ -130,4 +130,44 @@ - (void)testBytesAreDecodable {
   pb_release(gdt_cct_BatchedLogRequest_fields, &decodedBatch);
 }
 
+/** Tests that creating a message above the apparent threshold of 16320 bytes works. */
+- (void)testEncodingProtoAboveDefaultOSThreshold {
+  NSBundle *testBundle = [NSBundle bundleForClass:[self class]];
+  NSArray *testData = @[
+    @"message-32347456.dat", @"message-35458880.dat", @"message-39882816.dat",
+    @"message-40043840.dat", @"message-40657984.dat"
+  ];
+  NSMutableSet *storedEvents = [[NSMutableSet alloc] init];
+  // 250 messages results in a total size of 16337 which is > 16320, the apparent OS limit.
+  for (int i = 0; i < 250; i++) {  // Changing to 249 would've caused test to pass previously.
```

Comment:
Consider pulling this comment up a line to make it a continuation of the comment on the prior line.

Reply:
Done.